rasa/core/training/dsl.py

```diff
@@ -466,7 +499,11 @@ def add_event(self, event_name, parameters):
         # add 'name' only if event is not a SlotSet,
         # because there might be a slot with slot_key='name'
         if "name" not in parameters and event_name != SlotSet.type_name:
-            parameters["name"] = event_name
+            from rasa.nlu.training_data.formats.markdown import MarkdownReader
+
+            action_as_message = MarkdownReader().parse_e2e_training_example(event_name)
+            parameters["name"] = action_as_message.text
+            parameters["message"] = action_as_message
```

Comment:
@evgeniiaraz Why do we need this? Can't we just have an `ActionExecuted` where the action `name` equals the given text? Plus additionally maybe a flag that this is end-to-end?